dev-tools/xtask/src/a4x2_package.rs

```diff
@@ -0,0 +1,382 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Subcommand: cargo xtask a4x2-package
+
+use anyhow::Result;
+use camino::{Utf8Path, Utf8PathBuf};
+use clap::Parser;
+use serde::Deserialize;
+use sha2::Digest;
+use std::env;
+use std::fs;
+use std::io::{Read, Write};
+use walkdir::WalkDir;
+use xshell::{cmd, Shell};
+
+#[derive(Parser)]
+pub struct A4x2PackageArgs {
+    /// Bundle omicron live tests into the output tarball
+    #[clap(long)]
+    live_tests: bool,
+
+    /// Bundle omicron end-to-end-tests package into the output tarball
+    #[clap(long)]
+    end_to_end_tests: bool,
```

Comment:
Do you have a sense of how much more time these add?  I'd be inclined to make them on by default.